website/versioned_docs/version-5.x/testing/units.md

```diff
@@ -0,0 +1,109 @@
+
+An example of this can be found in `people` and `planets` applications from `react.microfrontends.app`. The [People application](https://github.com/react-microfrontends/people/blob/master/src/react-mf-people.js#L21) exports a function that resolves with a component. The [Planets Application](https://github.com/react-microfrontends/planets/blob/main/src/planets-page/selected-planet/selected-planet.component.js) imports and uses that component asynchronously with `React.lazy`. Testing this component would necessitate mocking both `SystemJS` and `People`.
+
+## Shared Mocks
+
+If each project mocks every other microfrontend it is possible that the mocks will eventually become out of sync with the actual deployed microfrontend. One way to prevent this is to share mocks so that keeping the mocks in sync only requires one change instead of updating mocks in many different repositories.
+
+## Testing examples
+
+### Jest
+In the above examples I showed how `People` imports a component from `styleguide`. In order to unit test the component in people with Jest you will need to [configure](https://jestjs.io/docs/configuration) jest and mock the `styleguide` MFE. In jest configuration is done via multiple areas.
+1. [Create a jest config file](#jest-config)
+1. [Setup a mocks directory at the root](#mocks-directory)
+1. [Add a setupFile](#setup-file)
+
+#### Jest config
+Jest is configured with a configuration file. Jest supports many options of configuration, below is an example configuration using some of the options. See [more options on Jest's official documentation site](https://jestjs.io/docs/configuration).
+````js
+module.exports = {
+  collectCoverageFrom: ['src/**/*.js'],
+  modulePathIgnorePatterns: ['/cache', '/dist'],
+  transform: {
+    '^.+\\.(j|t)sx?$': 'babel-jest',
+  },
+  setupFilesAfterEnv: ['./jest.setup.js'],
+  moduleNameMapper: {
+    // Note this is only needed if you don't match the module name directly
+    // an alternative would be to place the mock in 
+    // <rootDir>/__mocks__/@react-mf/styleguide.js and it would be autodetected
+    '@react-mf/styleguide': '<rootDir>/__mocks__/styleguide.js',
+  },
+}
+````
+
+#### mocks directory
+Jest will detect folders named `__mocks__` and if the naming convention is exact or the modules have been mapped using `moduleNameMapper` then Jest will use those mocks in place of an import. This structure is essential for other microfrontends where you don't have the code locally. [See more information on jest's official documentation](https://jestjs.io/docs/manual-mocks)
+```
+.
+├── __mocks__
+│   └── styleguide.js
+├── src
+│   ├── react-mf-people.js
+│   └── ...
+├── node_modules
+├── jest.setup.js
+├── ...
+└── jest.config.json
```

Comment:
I think this should be `jest.config.js`, since the example file above is a CJS file, not json.